lib/utils/cli.go

```diff
@@ -117,8 +117,10 @@ var initTestLoggerOnce = sync.Once{}
 // InitLoggerForTests initializes the standard logger for tests.
 func InitLoggerForTests() {
 	initTestLoggerOnce.Do(func() {
-		// Parse flags to check testing.Verbose().
-		flag.Parse()
+		if !flag.Parsed() {
+			// Parse flags to check testing.Verbose().
+			flag.Parse()
+		}
```

Comment:
The flow is evaluated in initTestLoggerOnce using sync.Once, which ensures that flag.Parse is called only once. 

How the `flag.Parse()` can be called twice if it wrapped sync.Once ?